python/requirements_compiled_py37.txt

```diff
@@ -0,0 +1,562 @@
+fastapi==0.99.1
+fasteners==0.18
+fastjsonschema==2.18.0
+feather-format==0.4.1
+ffmpy==0.3.1
+filelock==3.12.2
+flake8==3.9.1
+flake8-bugbear==21.9.2
+flake8-comprehensions==3.10.1
+flake8-quotes==2.0.0
+flaml==1.1.1
+flask==2.1.3
+flask-cors==4.0.0
+flatbuffers==2.0.7
+fonttools==4.38.0
+fqdn==1.5.1
+freezegun==1.1.0
+frozenlist==1.3.3
+fsspec==2023.1.0
+future==0.18.3
+gast==0.4.0
+gcs-oauth2-boto-plugin==3.0
+gitdb==4.0.10
+gitpython==3.1.32
+glfw==2.6.2
+glom==22.1.0
+google-api-core==2.11.1
+google-api-python-client==2.65.0
+google-apitools==0.5.32
+google-auth==2.22.0
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.6
+google-cloud-core==2.3.3
+google-cloud-storage==2.5.0
+google-crc32c==1.5.0
+google-pasta==0.2.0
+google-reauth==0.1.1
+google-resumable-media==2.5.0
+googleapis-common-protos==1.60.0
+gpustat==1.1
+gpy==1.10.0
+gpytorch==1.8.1
+gradio==3.11.0 ; platform_system != "Windows"
+grapheme==0.6.0
+graphql-core==3.2.3
+graphviz==0.20.1
+greenlet==2.0.2
+grpcio==1.56.2 ; sys_platform != "darwin"
+gsutil==5.25
+gunicorn==20.1.0
+gymnasium==0.28.1
+h11==0.12.0
+h5py==3.7.0
+heapdict==1.0.1
+hebo @ git+https://github.com/huawei-noah/HEBO@9a2a674c22518eed35a8b98e5134576741a95410#subdirectory=HEBO
+higher==0.2.1
+hjson==3.1.0
+hpbandster==0.7.4
+httpcore==0.15.0
+httplib2==0.20.4
+httptools==0.6.0
+httpx==0.24.1
+huggingface-hub==0.16.4
+humanfriendly==10.0
+hyperopt==0.2.7
+idna==3.4
+imageio==2.31.1
+imageio-ffmpeg==0.4.5
+imagesize==1.4.1
+importlib-metadata==4.10.0
+importlib-resources==5.12.0
+iniconfig==2.0.0
+ipykernel==6.16.2
+ipython==7.34.0
+ipython-genutils==0.2.0
+ipywidgets==7.8.0
+isodate==0.6.1
+isoduration==20.11.0
+isort==5.10.1
+itsdangerous==2.1.2
+jax-jumpy==1.0.0
+jedi==0.19.0
+jinja2==3.0.3
+jmespath==1.0.1
+joblib==1.2.0
+jschema-to-python==1.2.3
+json5==0.9.14
+jsondiff==2.0.0
+jsonpatch==1.32
+jsonpickle==3.0.1
+jsonpointer==2.4
+jsonschema==4.17.3
+jsonschema-spec==0.1.6
+junit-xml==1.9
+jupyter-cache==0.4.3
+jupyter-client==7.4.9
+jupyter-core==4.12.0
+jupyter-events==0.6.3
+jupyter-server==1.24.0
+jupyter-server-fileid==0.9.0
+jupyter-server-mathjax==0.2.6
+jupyter-server-ydoc==0.6.1
+jupyter-sphinx==0.3.2
+jupyter-ydoc==0.2.5
+jupyterlab==3.6.1
+jupyterlab-pygments==0.2.2
+jupyterlab-server==2.24.0
+jupyterlab-widgets==1.1.5
+jupytext==1.13.6
+kaggle-environments==1.7.11
+keras==2.11.0
+kiwisolver==1.4.4
+knack==0.10.1
+kubernetes==24.2.0
+labmaze==1.0.6
+lazy-object-proxy==1.9.0
+libclang==16.0.6
+lightgbm==3.3.5
+linkify-it-py==1.0.3
+llvmlite==0.39.1 ; python_version < "3.11"
+locket==1.0.0
+lxml==4.9.1
+lz4==4.3.2
+mako==1.2.4
+markdown==3.4.4
+markdown-it-py==1.1.0
+markupsafe==2.1.3
+matplotlib==3.5.3
+matplotlib-inline==0.1.6
+mccabe==0.6.1
+mdit-py-plugins==0.2.8
+memray==1.9.1 ; platform_system != "Windows" and sys_platform != "darwin" and platform_machine != "aarch64"
+minigrid==2.1.1
+mistune==0.8.4
+mlagents-envs==0.28.0
+mlflow==1.30.0 ; python_version <= "3.7"
```

Comment:
these `python_version` constraints can be removed?

Reply:
This is auto-added by pip-compile